amplify/backend/function/graphQlLambdaAuthorizerc6ebb49e/src/index.js

```diff
@@ -244,6 +251,13 @@ async function checkAuthorization(ast, variables, userInfo) {
         }
         break;
       }
+      case "deleteEvent": {
+        const eventId = retrieveArgument("eventId", selection.arguments);
```

Comment:
Just FYI, it looks like we are not sending `variables` in some of the `retrieveArgument` calls. Will address in future PR.

Reply:
Sounds good.